_playwright-tests/tests/API/Repositories.spec.ts

```diff
@@ -734,4 +734,69 @@ test.describe('Repositories', () => {
       expect(updatedRepo.gpgKey).toBe('test-gpg-key');
     });
   });
+
+  test('CRUD honours OrgId', async ({ client, cleanup }) => {
+    const expectedOrgId = '99999';
+    const repoName = `crud-orgid-test-${randomName()}`;
+    const repoUrl = randomUrl();
+    const updatedArch = 's390x';
+
+    await cleanup.runAndAdd(() => cleanupRepositories(client, repoName, repoUrl));
+
+    await test.step('Log the Org Id of the user', async () => {
+      console.log(`Testing CRUD operations with expected Org ID: ${expectedOrgId}`);
+    });
+
+    let repo: ApiRepositoryResponse;
+    await test.step('Create repo', async () => {
+      repo = await new RepositoriesApi(client).createRepository({
+        apiRepositoryRequest: {
+          name: repoName,
+          url: repoUrl,
+          distributionArch: 'x86_64',
+          distributionVersions: ['8'],
+        },
+      });
+      expect(repo.name).toBe(repoName);
+      expect(repo.url).toBe(repoUrl);
+      expect(repo.orgId).toBe(expectedOrgId);
+      console.log(`Repository created with Org ID: ${repo.orgId}`);
```

Comment:
i dont think the console.logs are necessary. if the org ID is not what we expect, we should see that in the test failure logs

Reply:
OK, removed. Thank you.